src/config/index.js

```diff
@@ -26,6 +27,8 @@ const configuration = {
   CUSTOMER_SUPPORT_EMAIL: 'customersuccess@edx.org',
   TABLEAU_URL: process.env.TABLEAU_URL,
   ENTERPRISE_LEARNER_PORTAL_URL: process.env.ENTERPRISE_LEARNER_PORTAL_URL,
+  LOGO_URL: process.env.LOGO_URL,
+  LOGO_WHITE_URL: process.env.LOGO_WHITE_URL,
```

Comment:
nit: might include `LOGO_TRADEMARK_URL` in here too so it's accessible if/when needed.

Reply:
Yeah, makes sense.